depends/Makefile

```diff
@@ -17,7 +18,7 @@ lib/libbigfile-mpi.a: bigfile/src/bigfile-mpi.c
 	cd bigfile/src; \
 	make install PREFIX=$(PWD) CC="$(MPICC)" MPICC="$(MPICC)" CFLAGS="$(OPTIMIZE)"
 
-lib/libpfft_omp.a:
+pfft:
```

Comment:
Ah. Good catch.

Reply:
Thanks! It was really Yueying who found it. Unfortunately I needed an extra commit to tell Make not to keep trying to create this file...